json/2.1/modules/ROOT/pages/json-module-troubleshooting.adoc

```diff
@@ -0,0 +1,55 @@
+= Troubleshooting JSON Module - Mule 4
+
+To troubleshoot JSON module, become familiar with the information about enabling verbose logging and interpreting commonly thrown messages.
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with the JSON module is failing, temporarily enable verbose logging for the module. +
+
+Remember to always disable the enhanced verbosity after troubleshooting, because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. Add an `<AsyncLogger>` tag inside the `<Loggers>` tag:
++
+[source,xml,linenums]
+----
+			<Loggers>
+				...
+				<AsyncLogger name="org.mule.module.json" level="DEBUG"/>
+				...
+			</Loggers>
+----
+[start=6]
+. Save your application changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+
+
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them:
+
+* JSON:INVALID_INPUT_JSON
+
+	The input document is not a valid JSON.
+
+* JSON:INVALID_SCHEMA
+
+ The supplied schema is invalid.
+
+* JSON:SCHEMA_NOT_FOUND
+
+ The schema could not be found.
+
+* JSON:SCHEMA_NOT_HONOURED
+
+ The input JSON document does not honour its schema.
```

Comment:
"honour" is a funny word here, though I have seen it before (and the American version is "honor"). 
Suggestion: The input JSON document does not adhere to its schema.